content/utmt/rooms/creating new rooms/_index.md

```diff
@@ -8,15 +8,39 @@ resources:
   - name: optimalRoomSide
     src: "optimalRoomSide.png"
     title: Optimal placing for left and right transitions
+  - name: RoomSetup
+    src: "RoomSetup.png"
+    title: This is how it should look now
```

Comment:
IMO "This is approximately how the room should look after the preparation" sounds+looks better as an alt